tribe/events/blocks/event-links.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Block: Event Links
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/blocks/event-links.php
+ *
+ * @package Nightingale
+ * @copyright NHS Leadership Academy, Andrew Blane
+ * @version 1.0 20th January 2021
+ *
+ */
+
+// don't show on password protected posts
+if ( post_password_required() ) {
+	return;
+}
+
+$has_google_cal = $this->attr( 'hasGoogleCalendar' );
+$has_ical       = $this->attr( 'hasiCal' );
+
+$should_render = $has_google_cal || $has_ical;
+
+remove_filter( 'the_content', 'do_blocks', 9 );
+
+if ( $should_render ) :
+?>
+	<div class="tribe-block tribe-block__events-link">
+		<?php if ( $has_google_cal ) : ?>
+			<div class="tribe-block__btn--link tribe-block__events-gcal">
+				<a
+					href="<?php echo Tribe__Events__Main::instance()->esc_gcal_url( tribe_get_gcal_link() ); ?>"
+					title="<?php esc_attr_e( 'Add to Google Calendar', 'the-events-calendar' ); ?>"
```

Comment:
can the domain be nightingale please, as part of theme files not plugin files.

Reply:
Ah yes. Done.